sdk/containerservice/arm-containerservicefleet/review/arm-containerservicefleet-node.api.md

```diff
@@ -484,14 +594,22 @@ export enum KnownUpdateState {
 export enum KnownUpgradeChannel {
     NodeImage = "NodeImage",
     Rapid = "Rapid",
-    Stable = "Stable"
+    Stable = "Stable",
+    TargetKubernetesVersion = "TargetKubernetesVersion"
 }
 
 // @public
 export enum KnownVersions {
+    V20220902Preview = "2022-09-02-preview",
+    V20230315Preview = "2023-03-15-preview",
+    V20230615Preview = "2023-06-15-preview",
+    V20230815Preview = "2023-08-15-preview",
     V20231015 = "2023-10-15",
+    V20240202Preview = "2024-02-02-preview",
     V20240401 = "2024-04-01",
-    V20250301 = "2025-03-01"
+    V20240502Preview = "2024-05-02-preview",
+    V20250301 = "2025-03-01",
+    V20250401Preview = "2025-04-01-preview"
```

Comment:
why so many preview added? could you share the spec pr with me?

Reply:
https://github.com/Azure/azure-rest-api-specs/pull/35829/files#diff-6df031bc2d45480b3c4504d438c389f559aca98f634eb70c3ac46a1fb12ddb33

this may because tcgc gives all the preview versions if the latest version is a previous version